images/nginx/rootfs/build.sh

```diff
@@ -18,13 +18,13 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-export NGINX_VERSION=1.19.10
+export NGINX_VERSION=1.21.4
```

Comment:
question: should we stick with v1.21.4 even 1.21.6 existing? should we cherry pick 21.6 patches?

Reply:
As Openresty hasn't been updated to 1.21.6, although not much has changed between 1.21.4 and 1.21.6. But I want to merge this first, to test this. Then update to 1.21.6 in subsequent PRs

/unhold